.github/workflows/build.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
       - name: Install dependencies
-        uses: bahmutov/npm-install@1a235c31658a322a3b024444759650ee6345c26d # tag=v1
+        uses: bahmutov/npm-install@1a235c31658a322a3b024444759650ee6345c26d # v1
```

Comment:
or i guess we can use `@v1`  instead

Reply:
Then Dependabot would just create a PR to pin them (like #18866)

Reply:
Maybe removing https://github.com/search?q=repo%3Aappium%2Fappium%20pinGitHubActionDigests&type=code would help...?
i haven't configured renovate well but maybe the helper does the work...?
https://docs.renovatebot.com/modules/manager/github-actions/

Reply:
This can of course be done, but is it not better to keep them pinned?

Reply:
I personally get annoyed by these renovate messages. I am ok to keep actions manually pinned to a specific tag like we do in other projects where we don't use renovate

Reply:
Hmm, fair enough - updating only on major tag changes would still be fine. In that case I will close this PR and create a new one that removes the digests from all actions, and adjusts the Renovate config to not pin them.